src/main/java/com/example/roster10/StaffController.java

```diff
@@ -46,4 +47,8 @@ public ResponseEntity<Object> deleteStaffById(@PathVariable("id") int id) {
         staffService.deleteStaffById(id);
         return ResponseEntity.ok(new StaffResponse("Staff deleted"));
     }
+    @GetMapping("/staff/")
```

Comment:
問題はないと思うのですが気になったので...
```suggestion
    @GetMapping("/staff")
```

Reply:
直しました。